strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/repository/NugetRepositoryManagementStrategy.java

```diff
@@ -97,4 +103,11 @@ private void createRemoteFeedDownloaderCronTask(String storageId,
             throw new RepositoryManagementStrategyException(e.getMessage(), e);
         }
     }
+    
+    public static boolean shouldDownloadRemoteRepositoryFeed()
```

Comment:
@carlspring I have created a separate property for Nuget as requested

Reply:
Thanks a lot! :)
Could you please update the [Application Properties](https://github.com/strongbox/strongbox/wiki/Application-Properties) page as necessary? Thanks! :)